apps/public-docsite-v9/src/Concepts/Migration/FromV0/Components/Flex.stories.mdx

```diff
@@ -0,0 +1,111 @@
+## Overview:
+
+Before:
+
+```tsx
+import { Flex } from '@fluentui/react-northstar';
+const Component = () => <Flex />;
+```
+
+After:
+
+```tsx
+import { Flex } from '@fluentui/react-components';
+const Component = () => <Flex />;
+```
+
+## How to migrate props:
+
+| `Flex` props              | migrate guide                                                              |
+| ------------------------- | -------------------------------------------------------------------------- |
+| className                 | keep it as is                                                              |
+| content                   | see [Migrate content prop](#./Migrate-`content`-prop) in this document     |
+| variables, design, styles | see [Migrate style overrides](#./Migrate-style-overrides) in this document |
+| debug                     | not supported, use custom style overrides if needed                        |
+| column                    | keep it as is                                                              |
+| fill                      | keep it as is                                                              |
+| gap                       | keep it as is                                                              |
+| hAlign                    | keep it as is                                                              |
+| inline                    | keep it as is                                                              |
+| padding                   | keep it as is                                                              |
+| space                     | keep it as is                                                              |
+| vAlign                    | keep it as is                                                              |
+| wrap                      | keep it as is                                                              |
+
+---
+
+## Migrate `content` prop
+
+Move `content` to JSX children.
+
+Before:
+
+```tsx
+import { Flex } from '@fluentui/react-northstar';
+const Component = () => <Flex content="hi" />;
+```
+
+After:
+
+```tsx
+import { Flex } from '@fluentui/react-components';
+const Component = () => <Flex>hi</Flex>;
+```
+
+## Migrate style overrides
+
+> Note: **If this is your first migration**, please read [the general guide on how to migrate styles](?path=/docs/concepts-migration-from-v0-custom-style-overrides--page). Also check examples in ["how to migrate styles" for Box component](./migrate-Box.md#Migrate-style-overrides).
+
+### Example for migrate boolean `variables`:
+
+Before:
+
+```tsx
+// in COMPONENT_NAME.tsx
+import { Flex } from '@fluentui/react-northstar';
+
+export const Component = () => <Flex variables={{ isSomething: true }} />;
+
+// in flex-styles.ts
+export const flexStyles = {
+  root: ({ variables: { isSomething } }) => ({
+    ...(isSomething && { color: 'red' }),
+  }),
+};
+```
+
+After:
+
+```tsx
+// in COMPONENT_NAME.tsx
+import { Flex } from '@fluentui/react-components';
```

Comment:
should we modify the package name?

Reply:
Done, replaced by `@fluentui/react-migration-v0-v9`